en/sensor/vectornav.md

```diff
@@ -0,0 +1,139 @@
+
+
+PX4 driver supports [all VectorNav sensors](https://www.vectornav.com/store/products).
+In particular the following systems are recommended:
+
+- **VN-200 GNSS/INS:** Recommended for fixed-wing systems without hovering, where static heading is not necessary.
+- **VN-300 DUAL GNSS/INS:** Recommended for multicopter systems where hovering and low dynamics requires the use of static heading.
+
+## Where to Buy
+
+VectorNav IMU/AHRS, GNSS/INS, and Dual GNSS/INS solutions are available directly from [VectorNav Technologies](https://www.vectornav.com/store/products) (US) or through their Global Sales Representatives. For more information on their solutions or for international orders, please contact sales@vectornav.com.
+
+[Purchase VN-200 Development Kit](https://www.vectornav.com/store/products/gnss-ins/p/vn-200-rugged-development-kit) (GNSS/INS)
+[Purchase VN-300 Development Kit](https://www.vectornav.com/store/products/dual-gnss-ins/p/vn-300-rugged-development-kit) (Dual GNSS/INS)
+
+
+
+## Hardware Setup
+
+### Wiring
+
+Connect any unused flight controller serial interface, such as a spare `GPS` or `TELEM` port, to the VectorNav UART2 port (required by PX4).
+
+### Mounting
+
+The VectorNav sensor can be mounted in any orientation, in any position on the vehicle, without regard to center of gravity.
+All VectorNav sensors default to a coordinate system of x-forward, y-right, and z-down, making the default mounting as connector-back, base down.
+This can be changed to any rigid rotation using the VectorNav Reference Frame Rotation register.
+
+If using a GNSS-enabled product, the GNSS antenna must be mounted rigidly with respect to the inertial sensor and with an unobstructed sky view. If using a dual-GNSS-enabled product (VN-3X0), the secondary antenna must be mounted rigidly with respect to the primary antenna and the inertial sensor with an unobstructed sky view.
+
+For more mounting requirements and recommendations, see the relevant [Quick Start Guide](https://www.vectornav.com/resources/quick-start-guides).
+
+## Firmware Configuration
+
+### PX4 Configuration
+
+To use the VectorNav driver:
+
+1. Include the module in firmware in the [kconfig board configuration](../hardware/porting_guide_config.md#px4-board-configuration-kconfig) by setting the kconfig variables: `CONFIG_DRIVERS_INS_VECTORNAV` or `CONFIG_COMMON_INS`.
+1. [Set the parameter](../advanced_config/parameters.md) [SENS_VN_CFG](../advanced_config/parameter_reference.md#SENS_VN_CFG) to the hardware port connected to the VectorNav (for more information see [Serial Port Configuration](../peripherals/serial_configuration.md)).
+1. Disable magnetometer preflight checks by setting [SYS_HAS_MAG](../advanced_config/parameter_reference.md#SYS_HAS_MAG) to `0`.
+1. Allow VectorNav driver to initialize by restarting PX4 with [SENS_VN_CFG](../advanced_config/parameter_reference.md#SENS_VN_CFG) set to nondefault.

```

Comment:
I'll remove "with [SENS_VN_CFG](../advanced_config/parameter_reference.md#SENS_VN_CFG) set to nondefault." - it is already set in previous step, so this is irrelevant.